Android/dokit/src/main/java/com/didichuxing/doraemonkit/kit/webdoor/bean/FpsJsbridgeBean.kt

```diff
@@ -1,28 +1,56 @@
 package com.didichuxing.doraemonkit.kit.webdoor.bean
 
+import android.os.Build
+import androidx.annotation.RequiresApi
 import com.didichuxing.doraemonkit.database.FpsEntity
+import com.didichuxing.doraemonkit.database.MemoryEntity
 
 data class FpsJsbridgeBean(
```

Comment:
Class name is inconsistent with the fields. Why add network, launch time, memory, location data here?

Reply:
has solved relevant suggestions, thank you